Content.Shared/Chemistry/EntitySystems/SharedHypospraySystem.cs

```diff
@@ -0,0 +1,74 @@
+using Content.Shared.Chemistry.Components;
+using Content.Shared.Timing;
+using Content.Shared.Verbs;
+using Content.Shared.Popups;
+using Robust.Shared.Player;
+using Content.Shared.Administration.Logs;
+
+namespace Content.Shared.Chemistry.EntitySystems;
+
+public abstract class SharedHypospraySystem : EntitySystem
+{
+    [Dependency] protected readonly UseDelaySystem _useDelay = default!;
+    [Dependency] protected readonly SharedPopupSystem _popup = default!;
+    [Dependency] protected readonly SharedSolutionContainerSystem _solutionContainers = default!;
+    [Dependency] protected readonly IEntityManager _entMan = default!;
```

Comment:
entitymanager is already a dependency on systems.

Reply:
remove _entMan and use EntityManager its not needed